recipes/pycvodes/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pycvodes" %}
+{% set version = "0.10.5" %}
+{% set sha256 = "5c751e92661f1fd032856adda34667f6e8b3edf181222fed8d2070621c2e590c" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 200
+  skip: True  # [win]
+  features:
+    - blas_{{ variant }}  # [not win]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
@bjodah, can you modify the recipe to use conda-build=3 stuff? Here's an experimental script https://gist.github.com/isuruf/a4c4c0a0c49114bc02cd8554b5d3b1f7

Reply:
Are there any examples of how `conda_build_config.yaml` should look like on conda-forge?
For example, does this look alright?:
```
# Pinning packages
python:
    - 2.7
    - 3.5
    - 3.6
numpy:
    - 1.11
openblas:
    - 0.2.20
```
(and is the `# Pinning packages` in the right place for use with your script?)

Reply:
Use the config file from `conda-forge-pinning`